### PR TITLE
update doc for split region

### PR DESCRIPTION
**Pull request by crazycs520**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update `sql-statements/sql-statement-split-region.md`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

N/A

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by crazycs520** [comment]
@nolouch PTAL

File: sql-statements/sql-statement-split-region.md [hunk 1]
```diff
@@ -416,6 +416,10 @@ region3:   [ 2<<61     ,  3<<61 )
 region4:   [ 3<<61     ,  +inf  )
 ```
 
+## 注意事项
+
+Split Region 语句切分的 Region 超过配置项 `split-merge-interval` 时间后，如果 Region 的 size 小于配置项 `max-merge-region-size` 的值，或者 Region 中的 key 的数量小于配置项 `max-merge-region-keys` 的值时，该 Region 可能会和相邻的 Region 合并。相关配置项的介绍和设置可以参考 [PD Control 使用说明](pd-control.md)
```
Comment (nolouch) [inline sql-statements/sql-statement-split-region.md]:
```suggestion
Split Region 语句切分的 Region 超过在 PD 上的配置项 `split-merge-interval` 时间后（默认 1h），且如果新分裂出来的 Region 的 size 小于配置项 `max-merge-region-size` 的值，或者 Region 中的 key 的数量小于配置项 `max-merge-region-keys` 的值时，该 Region 可能会和相邻的 Region 合并。相关配置项的介绍和设置可以参考 [PD Control 使用说明](pd-control.md)
```

File: sql-statements/sql-statement-split-region.md [hunk 2]
```diff
@@ -416,6 +416,10 @@ region3:   [ 2<<61     ,  3<<61 )
 region4:   [ 3<<61     ,  +inf  )
 ```
 
+## 注意事项
+
+Split Region 语句切分的 Region 超过在 PD 上的配置项 `split-merge-interval` 时间后（默认 1h），且如果新分裂出来的 Region 的 size 小于配置项 `max-merge-region-size` 的值，或者 Region 中的 key 的数量小于配置项 `max-merge-region-keys` 的值时，该 Region 可能会和相邻的 Region 合并。相关配置项的介绍和设置可以参考 [PD Control 使用说明](/pd-control.md)
```
Comment (yikeke) [inline sql-statements/sql-statement-split-region.md:421]:
```suggestion
Split Region 语句切分 Region 的时间超过 PD 配置项 [`split-merge-interval`](pd-configuration-file.md#split-merge-interval) 时间后（默认 1h），如果新分裂出来的 Region 出现以下任一情况，该 Region 可能会和相邻的 Region 合并：

- Region 的大小小于配置项 [`max-merge-region-size`](pd-configuration-file.md#max-merge-region-size) 的值
- Region 中的 key 的数量小于配置项 [`max-merge-region-keys`](pd-configuration-file.md#max-merge-region-keys) 的值时

相关配置项的设置方式可参考 [PD Control 使用说明](/pd-control.md)。
```

File: sql-statements/sql-statement-split-region.md [hunk 3]
```diff
@@ -416,6 +416,10 @@ region3:   [ 2<<61     ,  3<<61 )
 region4:   [ 3<<61     ,  +inf  )
 ```
 
+## 注意事项
```
Comment (yikeke) [inline sql-statements/sql-statement-split-region.md:419]: 建议标题具体些、有描述性些。比如：
```suggestion
## Split Region 与其他 Region 的合并情况
```

File: sql-statements/sql-statement-split-region.md (hunk 2 above)
Comment (crazycs520) [reply sql-statements/sql-statement-split-region.md:421]: @nolouch PTAL

不能去掉吧？应为不确定是否一定会合并，这处决于 PD 的调度

File: best-practices/pd-scheduling-best-practices.md [hunk 4]
```diff
@@ -103,6 +103,8 @@ Region 负载均衡调度主要依赖 `balance-leader` 和 `balance-region` 两
 
 Region merge 指的是为了避免删除数据后大量小甚至空的 Region 消耗系统资源，通过调度把相邻的小 Region 合并的过程。Region merge 由 `mergeChecker` 负责，其过程与 `replicaChecker` 类似：PD 在后台遍历，发现连续的小 Region 后发起调度。
 
+具体来说，当某个新分裂出来的 Region 超过配置项 [`split-merge-interval`](/pd-configuration-file.md#split-merge-interval) 时间后（默认 1h），且这个 Region 的 size 小于配置项 [`max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size) 的值（默认 20MiB），或者 Region 中的 key 的数量小于配置项 [`max-merge-region-keys`](/pd-configuration-file.md#max-merge-region-keys) 的值时（默认 200000），该 Region 会触发 Region merge 的调度。
```
Comment (crazycs520) [inline best-practices/pd-scheduling-best-practices.md:106]:
```suggestion
具体来说，当某个新分裂出来的 Region 超过配置项 [`split-merge-interval`](/pd-configuration-file.md#split-merge-interval) 时间后（默认 1h），且这个 Region 的 size 小于配置项 [`max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size) 的值（默认 20MiB），或者 Region 中的 key 的数量小于配置项 [`max-merge-region-keys`](/pd-configuration-file.md#max-merge-region-keys) 的值时（默认 200000），该 Region 会触发 Region merge 调度。
```

File: sql-statements/sql-statement-split-region.md (hunk 3 above)
Comment (crazycs520) [reply sql-statements/sql-statement-split-region.md:419]: em... 我还是倾向用 "注意事项"， 用 “Split Region 与其他 Region 的合并情况” 标题可能不会引起注意(⊙o⊙)…

**Comment by crazycs520** [comment]
@yikeke @nolouch PTAL

File: sql-statements/sql-statement-split-region.md (hunk 3 above)
Comment (yikeke) [reply sql-statements/sql-statement-split-region.md:419]: 下面内容改成给链接了，这里用「注意事项」也可

File: best-practices/pd-scheduling-best-practices.md [hunk 5]
```diff
@@ -103,6 +103,8 @@ Region 负载均衡调度主要依赖 `balance-leader` 和 `balance-region` 两
 
 Region merge 指的是为了避免删除数据后大量小甚至空的 Region 消耗系统资源，通过调度把相邻的小 Region 合并的过程。Region merge 由 `mergeChecker` 负责，其过程与 `replicaChecker` 类似：PD 在后台遍历，发现连续的小 Region 后发起调度。
 
+具体来说，当某个新分裂出来的 Region 超过配置项 [`split-merge-interval`](/pd-configuration-file.md#split-merge-interval) 时间后（默认 1h），且这个 Region 的 size 小于配置项 [`max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size) 的值（默认 20MiB），或者 Region 中的 key 的数量小于配置项 [`max-merge-region-keys`](/pd-configuration-file.md#max-merge-region-keys) 的值时（默认 200000），该 Region 会触发 Region merge 调度。
```
[the remark this answers is not in the document]
Comment (crazycs520) [reply best-practices/pd-scheduling-best-practices.md:106]: 嗯嗯，是的

[the remark this answers is not in the document]
Comment (yikeke) [reply best-practices/pd-scheduling-best-practices.md:106]:
```suggestion
具体来说，当某个新分裂出来的 Region 存在的时间超过配置项 [`split-merge-interval`](/pd-configuration-file.md#split-merge-interval) 的值后（默认 1h），且该 Region 大小小于配置项 [`max-merge-region-size`](/pd-configuration-file.md#max-merge-region-size) 的值（默认 20MiB），或者该 Region 中 key 的数量小于配置项 [`max-merge-region-keys`](/pd-configuration-file.md#max-merge-region-keys) 的值时（默认 200000），该 Region 会触发 Region merge 调度。
```

File: sql-statements/sql-statement-split-region.md [hunk 6]
```diff
@@ -416,6 +416,10 @@ region3:   [ 2<<61     ,  3<<61 )
...
+
+Split Region 语句切分的 Region 会受到 PD 中 Region Merge 调度的控制，需要避免新切分的 Region 不久后又被 PD 重新 Merge 的情况，详情请参考 [PD 调度策略最佳实践/Region Merge](/best-practices/pd-scheduling-best-practices.md#region-merge)。动态修改 Region Merge 相关的配置项，请参考 [PD Control 使用说明](/pd-control.md)。
```
Comment (yikeke) [inline sql-statements/sql-statement-split-region.md:421]:
```suggestion
Split Region 语句切分的 Region 会受到 PD 中 [Region Merge](/best-practices/pd-scheduling-best-practices.md#region-merge) 调度的控制，需要[动态修改](/pd-control.md) Region Merge 相关的配置项，避免新切分的 Region 不久后又被 PD 重新合并的情况。
```

**Review by nolouch — APPROVED** [review]

**Review by yikeke — APPROVED** [review]
/lgtm

**Pull request merged**